src/app/card/card.component.html

```diff
@@ -4,16 +4,16 @@
 
   <div class="card-content {{ textClass }}">
 
-    <span class="card-title">
+    <div class="card-title" *ngIf="hasCardTitle">
```

Comment:
It will not validate if the ng-content has something inside its inner HMTL. This means if we add an empty `<mz-car-title></mz-card-title>` the class `card-title` will be applied.